.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,110 @@
+name: "Bug report \U0001F41B"
```

```suggestion
name: "Bug report 🐛"
```
Let's keep items consistent